src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    if (!Animal.alive) {
+      Animal.alive = [];
+    }
```

Comment:
Why you can`t simply add?
```suggestion
      Animal.alive = [];
```

Why do you need this check?

Reply:
Because if i do it like:

`   Animal.alive = [];
    Animal.alive.push(this);`

It will create empty array every time when constructor is called

Reply:
it could be better to init `alive` using `static`, but then i had to disable linter

Reply:
Ok. I understand. With `static` it truly would be better. You have problems with linter if you add
`static alive = [];` in line 4, before constructor - too?

Reply:
Yes, it throws exception "Unexpected token ="